symphony/bdk/core/auth/auth_session.py

```diff
@@ -0,0 +1,30 @@
+class AuthSession:
+    """RSA Authentication session handle.
+    """
+
+    def __init__(self, authenticator):
+        self._session_token = None
+        self._key_manager_token = None
+        self._authenticator = authenticator
+
+    @property
+    def session_token(self) -> str:
```

Comment:
Maybe use properties only if we need to change the behavior of accessing a certain function.
So usually we won't use any properties for normally accessing an attribute